includes/frontend/class-wcal-checkout-process.php

```diff
@@ -530,8 +537,8 @@ public function wcal_order_placed( $order_id ) {
 					}
 				}
 
-				add_post_meta( $order_id, 'wcal_recover_order_placed_sent_id', $email_sent_id );
-				add_post_meta( $order_id, 'wcal_recover_order_placed', $abandoned_order_id );
+				$order->add_meta_data( 'wcal_recover_order_placed_sent_id', $email_sent_id );
+				$order->add_meta_data( $order_id, 'wcal_recover_order_placed', $abandoned_order_id );
```

Comment:
The parameters passed to the add_meta_data() are incorrect. Please remove `$order_id`